src/presets/seegno.json

```diff
@@ -1,74 +1,23 @@
 {
   "additionalRules": [ "../rules/*.js" ],
-  "disallowDanglingUnderscores": true,
-  "disallowEmptyBlocks": true,
-  "disallowGeneratorsInDescribeFunctions": true,
-  "disallowKeywords": [
-    "with"
-  ],
-  "disallowKeywordsInComments": true,
   "disallowKeywordsOnNewLine": [
     "catch",
     "else",
     "while"
   ],
-  "disallowMixedSpacesAndTabs": true,
-  "disallowMultipleLineBreaks": true,
-  "disallowMultipleLineStrings": true,
-  "disallowMultipleSpaces": true,
-  "disallowMultipleVarDecl": "strict",
   "disallowNewlineBeforeBlockStatements": true,
-  "disallowNodeTypes": [
-    "LabeledStatement"
-  ],
-  "disallowOnlyFilterInTestFunctions": true,
-  "disallowOperatorBeforeLineBreak": true,
-  "disallowPaddingNewlinesInBlocks": true,
-  "disallowQuotedKeysInObjects": true,
-  "disallowSpaceAfterObjectKeys": true,
-  "disallowSpaceAfterPrefixUnaryOperators": true,
-  "disallowSpaceBeforeComma": true,
-  "disallowSpaceBeforePostfixUnaryOperators": true,
-  "disallowSpaceBeforeSemicolon": true,
-  "disallowSpacesInCallExpression": true,
-  "disallowSpacesInFunctionDeclaration": {
-    "beforeOpeningRoundBrace": true
-  },
-  "disallowSpacesInsideArrayBrackets": true,
-  "disallowSpacesInsideParentheses": true,
-  "disallowSpacesInsideParenthesizedExpression": true,
-  "disallowSpacesInsideTemplateStringPlaceholders": true,
```

Comment:
Can't find this in the description of https://github.com/seegno/eslint-config-seegno/pull/45.